{"test_list": [{"style": "Error", "name": "test_json_test_runner.RunnerCase", "doc": "", "count": 3, "Pass": 1, "fail": 1, "error": 1, "cid": "c1", "info": [{"index": "test_json_test_runner.RunnerCase.test_error", "name": "test_error", "style": "Error", "desc": "", "script": "raise with error\nTraceback (most recent call last):\n  File \"/Users/work/json-test-runner/test/test_json_test_runner.py\", line 24, in test_error\n    raise Exception('this is error')\nException: this is error\n", "status": "错误"}, {"index": "test_json_test_runner.RunnerCase.test_failed", "name": "test_failed", "style": "Failed", "desc": "", "script": "1 != 2\nTraceback (most recent call last):\n  File \"/Users/work/json-test-runner/test/test_json_test_runner.py\", line 20, in test_failed\n    self.assertEqual(1, 2)\nAssertionError: 1 != 2\n", "status": "失败"}, {"index": "test_json_test_runner.RunnerCase.test_pass", "name": "test_pass", "style": "Pass", "desc": "\n        this is pass\n        :return:\n        ", "script": "1 == 1\n", "status": "通过"}]}, {"style": "Failed", "name": "test_json_test_runner1.RunnerTestCase", "doc": "", "count": 2, "Pass": 1, "fail": 1, "error": 0, "cid": "c2", "info": [{"index": "test_json_test_runner1.RunnerTestCase.test_failed", "name": "test_failed", "style": "Failed", "desc": "", "script": "Traceback (most recent call last):\n  File \"/Users/work/json-test-runner/test/test_json_test_runner1.py\", line 13, in test_failed\n    self.assertTrue(0)\nAssertionError: 0 is not true\n", "status": "失败"}, {"index": "test_json_test_runner1.RunnerTestCase.test_success", "name": "test_success", "style": "Pass", "desc": "", "script": "success\n", "status": "通过"}]}], "count": 5, "Pass": 2, "fail": 2, "error": 1, "passrate": "40.00%", "start_time": "2022-09-26 18:09:35", "stop_time": "2022-09-26 18:09:36", "duration": "0:00:01.003048"}